uasc/secure_channel.go

```diff
@@ -116,12 +115,12 @@ func (s *SecureChannel) hasState(n int32) bool {
 }
 
 // Send sends the service request and calls h with the response.
-func (s *SecureChannel) Send(svc interface{}, authToken *ua.NodeID, h func(interface{}) error) error {
+func (s *SecureChannel) Send(svc ua.Request, authToken *ua.NodeID, h func(interface{}) error) error {
```

Comment:
This should arguably be called `SendRequest` now

Reply:
Yes.